### PR TITLE
[3642] Use variable template for stacks-dotnet

### PR DESCRIPTION
**Pull request by russellseymour**
#### 📲 What

Move variables out of main build file into separate variable template file

#### 🤔 Why
		
By making the variables a separate file it is easier to automate its creation in the Stacks CLI
		
#### 🛠 How
		
All of the variables are written to the variable template file, `azuredevops-vars.yaml`. This is included in the main build file.

#### 👀 Evidence
		
The builds are working as before
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

**Comment by ElvenSpellmaker** [comment]
Hey Russell, just a quick one, should prod variables and nonprod go into separate files?

File: build/azDevOps/azure/azure-pipelines-netcore-k8s.yml [hunk 1]
```diff
@@ -25,9 +25,7 @@ resources:
     - repository: templates
       type: github
       name: amido/stacks-pipeline-templates
-      ref: refs/tags/v2.0.4
-      # EXCHANGE THIS FOR YOUR OWN ENDPOINT CONNECTION TO GITHUB
-      # REPOSITORY IS PUBLIC
+      ref: refs/tags/v2.0.1
```
Comment (ElvenSpellmaker) [inline build/azDevOps/azure/azure-pipelines-netcore-k8s.yml:28]: Is there a reason you've rolled back in the Pipeline Templates?

Comment (russellseymour) [reply build/azDevOps/azure/azure-pipelines-netcore-k8s.yml:28]: No that is a mistake which I will correct

**Pull request merged**